### ISSUE TITLE
index.html is missing reference to Weedopedia

### ISSUE THREAD

**Issue by jarednova** [enatario/snowflake#1]
Not all of Mr. Snowflake's work is represented on the [`index.html`](https://github.com/enatario/snowflake/blob/master/src/index.html) page.

### Steps to reproduce
Observe the index.html file contains books, but no reference to Weedopedia

### What we should see
We should see Weedopedia represented: both the book and the desk calendar

**Comment by enatario** [comment]
Good call. Weedopedia is central to Snowflake's authorship. Should the calendar and the book be separate items or represented under one title with references to each product?

**Comment by jarednova** [comment]
There are good business cases to be made either way. I think we should leave that to the lead stakeholder to make the final determination

**Comment by enatario** [comment]
👍
